content/hi/docs/home/_index.md

```diff
@@ -0,0 +1,65 @@
+---
+title: कुबेरनेट्स प्रलेखन  
+noedit: true
+cid: docsHome
+layout: docsportal_home
+class: gridPage gridPageHome
+linkTitle: "होम"
+main_menu: true
+weight: 10
+hide_feedback: true
+menu:
+  main:
+    title: "प्रलेखन "
```

Comment:
```suggestion
    title: "प्रलेखन"
```

Reply:
I am not sure if they changed their docs guidelines recently, but you can also try removing the `"`.